integrated_channels/utils.py

```diff
@@ -381,3 +382,26 @@ def is_valid_url(url):
         return all([result.scheme, result.netloc])
     except ValueError:
         return False
+
+
+def batch_by_pk(ModelClass, extra_filter=Q(), batch_size=10000):
```

Comment:
i wrote this helper and had copy/pasted it twice before. moving it into a utility class for re-use.